src/dev/MAX31855.cpp

```diff
@@ -0,0 +1,25 @@
+#include <EVT/io/SPI.hpp>
+#include <dev/MAX31855.h>
+
+using namespace EVT::core::IO;
+
+namespace DEV {
+
+MAX31855::MAX31855(SPI& spi, uint8_t dev) : spi(spi) {
+    this->device = dev;
+}
+
+int16_t MAX31855::readTemp() {
+    int16_t temp;
+    const int length = 4;
+    uint8_t bytes[length];
+    spi.startTransmission(this->device);
```

Comment:
```suggestion
    spi.startTransmission(device);
```
This is just personal preference